src/ui/server/controllers/insurance/your-buyer/validation/rules/countryValidationRules.test.ts

```diff
@@ -0,0 +1,26 @@
+import generateValidationErrors from '../../../../../helpers/validation';
+import { FIELDS } from '../../../../../content-strings/fields/insurance/your-buyer';
+import { ERROR_MESSAGES } from '../../../../../content-strings';
+import { FIELD_IDS } from '../../../../../constants';
+import { countryValidationRules } from './countryValidationRules';
+
+const { YOUR_BUYER } = FIELDS;
+
+describe('controllers/insurance/your-buyer/validation/country-rules/', () => {
```

Comment:
Excellent having the rules in their own files, but can we stick with the file naming convention, `/validation/rules/country.ts`

instead of camel case

Reply:
done